storage/BUILD

```diff
@@ -0,0 +1,24 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_docker//cc:image.bzl", "cc_image")
```

Comment:
Are you going to make a docker image? Otherwise, this is not needed.

Reply:
No, I will remove it.